src/main/java/com/example/graduationproject/Controller/UsersModelController.java

```diff
@@ -11,6 +11,7 @@
 
 @Controller
 public class UsersModelController {
```

Comment:
Controllers and services shouldn't contain constructors
Use @Autowired to inject dependencies

Reply:
done